popup/component/app.js

```diff
@@ -58,35 +65,65 @@ export default class App extends Component {
 
   render () {
     const { store } = this.props;
-    const { enabled, extractions } = this.state;
+    const { augmentationEnabled, extractions, status, url } = this.state;
 
     return (
       <div className={ styles.container }>
         <div className={ styles.header }>
           <h1 className={ styles.title }>Web3 Injection</h1>
```

Comment:
When nothing was extracted on the website, I would rephrase this to "No identities have been found on the page."